scripts/polarion_sync/scanner.py

```
@@ -0,0 +1,303 @@
+    is_std_only: bool = False
+    # Existing markers attached to the test (informational)
+    markers: list[str] = field(default_factory=list)
+    # Jira issue keys found in docstrings (test, class, module)
+    jira_ids: list[str] = field(default_factory=list)
+    # Polarion IDs from other tests in the same file (for requirement fallback)
+    sibling_polarion_ids: list[str] = field(default_factory=list)
+
+
+def _changed_test_files(repo_root: Path) -> list[Path]:
+    """Return ``test_*.py`` files added or modified in the last commit."""
+    result = subprocess.run(
```

Ruff (0.15.15)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)